### PR TITLE
copyright changes

### PR DESCRIPTION
**Pull request by hemalathagajendran**
## Pull request checklist

Copyright changes 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature Enhancement
- [ ] Test Automation 
- [ ] Code Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Community Operator listing 
- [x ] Other (please describe): copyright changes

## How risky is this change?
- [ x] Small, isolated change
- [ ] Medium, requires regression testing
- [ ] Large, requires functional and regression testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ We welcome contributions to this project, see [Contributing](CONTRIBUTING.md) fo
 
 ## Licensing
 
-Copyright 2022 IBM Corp.
+Copyright 2022, 2024 IBM Corp.
```
Comment (saurabhwani5) [inline README.md:31]: can we remove , from here

**Review by Jainbrt — DISMISSED** [review]

**Review by saurabhwani5 — DISMISSED** [review]

File: operator/controllers/suite_test.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022.
+Copyright 2022, 2024.
```
Comment (deeghuge) [inline operator/controllers/suite_test.go:2]: Change it to
Copyright 2022, 2024 IBM Corp.

File: operator/main.go [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022.
+Copyright 2022, 2024.
```
Comment (deeghuge) [inline operator/main.go:2]: Change it to
Copyright 2022, 2024 IBM Corp.

**Review by deeghuge — CHANGES_REQUESTED** [review]

File: operator/main.go (hunk 3 above)
Comment (hemalathagajendran) [reply operator/main.go]: Done

File: operator/controllers/suite_test.go (hunk 2 above)
Comment (hemalathagajendran) [reply operator/controllers/suite_test.go]: Done

File: driver/csiplugin/utils/utils.go [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016, 2019 IBM Corp.
+ * Copyright 2016, 2019, 2024 IBM Corp.
```
Comment (deeghuge) [inline driver/csiplugin/utils/utils.go:2]: we should remove 2019 from here

File: operator/api/v1/csiscaleoperator_types.go [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022.
+Copyright 2022, 2024.
```
Comment (deeghuge) [inline operator/api/v1/csiscaleoperator_types.go:2]: IBM Corp is missing here

File: operator/api/v1/groupversion_info.go [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022.
+Copyright 2022, 2024.
```
Comment (deeghuge) [inline operator/api/v1/groupversion_info.go:2]: IBM Corp is missing here

File: driver/cmd/ibm-spectrum-scale-csi/main.go [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2018, 2024 The Kubernetes Authors.
```
Comment (deeghuge) [inline driver/cmd/ibm-spectrum-scale-csi/main.go:2]: Change this to IBM Corp instead of Kubernetes Authors

**Review by deeghuge — CHANGES_REQUESTED** [review]

File: driver/csiplugin/utils/utils.go (hunk 4 above)
Comment (hemalathagajendran) [reply driver/csiplugin/utils/utils.go]: Done

File: operator/api/v1/csiscaleoperator_types.go (hunk 5 above)
Comment (hemalathagajendran) [reply operator/api/v1/csiscaleoperator_types.go]: Done

File: operator/api/v1/groupversion_info.go (hunk 6 above)
Comment (hemalathagajendran) [reply operator/api/v1/groupversion_info.go]: Done

File: driver/cmd/ibm-spectrum-scale-csi/main.go (hunk 7 above)
Comment (hemalathagajendran) [reply driver/cmd/ibm-spectrum-scale-csi/main.go]: Done

**Review by deeghuge — APPROVED** [review]

**Pull request merged**